client/components/Checkout.js

```diff
@@ -200,15 +183,13 @@ class Checkout extends Component {
                   </tr>
                 </thead>
                 <tbody>
-                  {orderItems.length
-                    ? orderItems.map(orderItem => (
-                        <tr key={orderItem.id}>
-                          <td>{findProduct(products, orderItem).title}</td>
-                          <td>${orderItem.price.toFixed(2)}</td>
-                          <td>{orderItem.quantity}</td>
-                          <td>
-                            ${orderItem ? (orderItem.price * orderItem.quantity).toFixed(2) : 0}
-                          </td>
+                  {order
```

Comment:
`order` will always be true

```suggestion
                  {order.orderItems.length
```